workflow/header/nsc_header.txt

```diff
@@ -0,0 +1,12 @@
+#parent_id: spp
+#parent_name: 'ChIP-seq processing pipeline'
+#id: 'nsc_coefficient'
+#section_name: 'NSC coefficient'
+#description: "generated using run_spp.R script from <a href='https://github.com/kundajelab/phantompeakqualtools' target='_blank'>phantompeakqualtools</a>."
+#plot_type: 'bargraph'
+#anchor: 'nsc_coefficient'
+#pconfig:
```

Comment:
There are three things that might help, but I am also just guessing:

1. Ensure there's a whitespace between the `#` and the the header lines (because that's the way the example does it;).
2. Explicitly set the `# file_format: 'tsv'` -- who knows which code path that activates.
3. Also make the `parent_id` a string, i.e. `# parent_id: 'spp'

So my suggestion here would be:
```suggestions
# parent_id: 'spp'
# parent_name: 'ChIP-seq processing pipeline'
# id: 'nsc_coefficient'
# section_name: 'NSC coefficient'
# description: "generated using run_spp.R script from <a href='https://github.com/kundajelab/phantompeakqualtools' target='_blank'>phantompeakqualtools</a>."
# plot_type: 'bargraph'
# anchor: 'nsc_coefficient'
# pconfig:
```

If this doesn't do the trick, another guess would be to try to prepend the `parent_id` to the section `id`, i.e.:
```
# id: 'spp-nsc_coefficient'
```

This seems to be what multiqc does internally, so maybe that helps... :D

Reply:
Ah, and maybe skipping the `# anchor:` line would help, because that might override the default anchor that multiqc wants to set...